full-stack-inspiration-board/submit.md

```diff
@@ -190,3 +190,28 @@ The first slide must include the team name and each member's name. The presentat
 ##### !end-answer
 ### !end-challenge
 <!-- prettier-ignore-end -->
+
+## Reflect on Pair Programming
```

Comment:
Should we rephrase this to be about a group rather than a pair?

Reply:
Inspo board and group fansite will always be at least 3 folks, so it makes sense to update the language on those, I'll push an update shortly.